public/app/features/explore/TraceView/components/TracePageHeader/NewTracePageHeader.tsx

```diff
@@ -200,5 +175,22 @@ const getNewStyles = (theme: GrafanaTheme2) => {
       max-width: 30%;
       display: inline-block;
     `,
+    tracePageHeaderBack: css`
+      label: TracePageHeaderBack;
+      align-items: center;
+      align-self: stretch;
+      background-color: #fafafa;
+      border-bottom: 1px solid #ddd;
+      border-right: 1px solid #ddd;
+      color: inherit;
+      display: flex;
+      font-size: 1.4rem;
+      padding: 0 1rem;
+      margin-bottom: -1px;
+      &:hover {
+        background-color: #f0f0f0;
+        border-color: #ccc;
+      }
+    `,
```

Comment:
@grafana/observability-traces-and-profiling maybe this is something you want to have a look at, it was ported from previous code but desn't look right

Reply:
Yeah this styling can be removed and the link placed next to the duration. It doesn't look great. I see you've already removed the className in TraceViewContainer so that the older trace view header (to be removed in the near future) is not using this styling. I can add an item to our backlog if you want to keep it out of this PR?

![Screenshot 2023-06-27 at 14 26 51](https://github.com/grafana/grafana/assets/90795735/8fed0d02-028b-4087-a26f-18adfe6dc778)